### DISCUSSION TITLE
🏦 | Database

### DISCUSSION THREAD

**Discussion by EhsanFox** [SeTar-Bot/Setar-Bot#61] [Q&A] [answered]
### [Questions related to Database Modules](https://github.com/SeTar-Bot/bot/tree/main/src/Database) 

👀 Make sure to tell us:
- What branch/version you are using
- What exactly are you trying to change or do
- Have you checked the MongoDB Connection

Also, don't forget to [Join our Discord Community](https://discord.gg/7jgfP6j4Tc)

**Comment by im-parsa** [comment, marked as answer]
Is it ok to edit the schema entities?
